src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
+	 *
+	 * @param Pages $pages
+	 *
+	 * @return bool
+	 *
+	 */
+	public function purgePages( Pages $pages ) {
+		$pagesArray = $pages->toArray();
+
+		foreach ( $pagesArray as $page ) {
+			$this->purge( $page );
+		}
+
+		return true;
```

Comment:
Yeah, I think checking the result of the request would be best (and, as you say, that won't matter if the request throws an exception). I guess it just means that if no pages were purged, `false` will be returned? Actually, there are three results possible for each page: `purged`, `missing`, and `invalid`. Perhaps the purge methods should return true only when *all* results are `purged` and false if any are the other two?